src/main/java/frc/robot/Constants.java

```diff
@@ -36,8 +36,8 @@
     
     public static final int kLeftJoystickAxisX = 0;
     public static final int kLeftJoystickAxisY = 1;
-    public static final int kRightJoystickAxisX = 4;
-    public static final int kRightJoystickAxisY = 5;
+    public static final int kRightJoystickAxisX = 2;
```

Comment:
Would be worth adding a comment about about what brand/type of controller these mapping work for.